handsontable/src/dataMap/metaManager/metaSchema.js

```diff
@@ -1840,6 +1840,102 @@ export default () => {
      */
     disableVisualSelection: false,
 
+    /**
+     * @description
+     * The `dialog` option configures the [`Dialog`](@/api/dialog.md) plugin.
+     *
+     * You can set the `dialog` option to one of the following:
+     *
+     * | Setting   | Description                                                                 |
+     * | --------- | --------------------------------------------------------------------------- |
+     * | `false`   | Disable the [`Dialog`](@/api/dialog.md) plugin                              |
+     * | `true`    | Enable the [`Dialog`](@/api/dialog.md) plugin with default options          |
+     *
+     * ##### dialog: Additional options
+     *
+     * | Option                   | Possible settings                                  | Description                                                                                                                                                      |
+     * | ------------------------ | -------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+     * | `content`                | A string or HTML element (default: `''`)           | The content of the dialog                                                                                                                                        |
+     * | `customClassName`        | A string (default: `''`)                           | The custom class name of the dialog                                                                                                                              |
+     * | `background`             | A string (default: `'semi-transparent'`)           | The background of the dialog                                                                                                                                     |
+     * | `contentBackground`      | Boolean (default: `true`)                          | Whether to show the content background                                                                                                                           |
```

Comment:
In the code, the default value is defined as `false`. Is it right?
```suggestion
     * | `contentBackground`      | Boolean (default: `false`)                          | Whether to show the content background                                                                                                                           |
```